app/controllers/responses_controller.rb

```diff
@@ -0,0 +1,20 @@
+class ResponsesController < ApplicationController
+
+  def new
```

Comment:
@jeder28 Now that we have user auth, can you add the necessary checks here? Thanks! :+1: